portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -26,11 +31,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     System.out.println("Button val: " + id);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Query query = new Query("Task").setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
```

Comment:
It would be better to make all the table name and column name as a separate constant class, which could be called in each servlet. Therefore, you won't have version conflicts issue in the future if you made any changes in the database strucutre.

Reply:
Sorry, I don't quite know what you mean. Do you mean that I should replace "Task", "id" literals with constants from a class?

Reply:
Yes I meant you could have a separate class defines constants for all those table/column name. In this way, you could use ConstantUtils.TABLE_TASK instead of using "Task" in each servlet (similarly ConstantUtils.COLUMN_ID instead of "id"). 
It is just a proposal so you don't need to change it now. Just a heads-up, in case of more complicated data storage structure (e.g. capstone project), it would be better to manage it this way to avoid potential conflicts.